src/main/scala/mesosphere/marathon/api/MarathonExceptionMapper.scala

```diff
@@ -18,11 +18,17 @@ import play.api.libs.json.{JsResultException, JsValue, Json}
 
 import scala.concurrent.TimeoutException
 
+/**
+  * Class used to map exceptions thrown in Jersey controller methods. It is configured by providing it to
+  * [[RootApplication]].
+  *
+  * Handles a few exceptions, plus all [[Rejection]]s via RejectionException
+  */
 @Provider
 @Singleton
 class MarathonExceptionMapper extends ExceptionMapper[JavaException] with StrictLogging {
 
-  def toResponse(exception: JavaException): Response = {
+  private def exceptionToResponse(exception: JavaException): Response = {
```

Comment:
What's the motivation of making it private?

Reply:
They are not used outside.

Reply:
So? I guess that's a longer discussions to have, though. I usually make every public by default unless it changes state and should not be exposed.